mars/base_app.py

```diff
@@ -185,16 +183,7 @@ def create_pool(self, *args, **kwargs):
         return create_actor_pool(*args, **kwargs)
 
     def main_loop(self):
-        if self.args.profile == -1:
```

Comment:
Why remove the profile?

Reply:
Profile config is useless in main process.